demo/setup.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
 set -e
+dir="$(dirname "${BASH_SOURCE[0]}")"
+: ${PYTHON:=python}
 
-if [[ ! -f ~/envs/plop-demo/bin/activate ]]; then
-    virtualenv -p python2.7 --no-site-packages ~/envs/plop-demo
+if [[ ! -f "$dir/envs/plop-demo/bin/activate" ]]; then
+    "$PYTHON" -mvirtualenv --no-site-packages "$dir/envs/plop-demo" || \
+        "$PYTHON" -mvenv "$dir/envs/plop-demo"
```

Comment:
The commit message claims to prefer `venv` if available, but this prefers `virtualenv`.

Python 3.3 `venv` did not install `pip` automatically, so `virtualenv` is probably the safer choice. I'd rather just require `virtualenv` than have the logic to switch between them.